### PR TITLE
tweak notation for better terminology

### PR DESCRIPTION
**Pull request by dhh1128**

### CODE REVIEW

**Review by swcurran — APPROVED** [review]
Hard to see all the changes, but what I've checked looks good.  Nice to have a LT;DR of the substantive changes (if any).

File: concepts/0006-ssi-notation/README.md [hunk 1]
```diff
@@ -51,334 +33,151 @@ This notation aims to be:
...
-other ad-hoc meanings). When reading aloud, the spoken form of
-a symbol like this is the name of the letter. The relevant [ABNF](
-https://tools.ietf.org/html/rfc5234) fragment is:
+When an identity controller controls its own identity, we say that it has __self sovereignty__ -- and we call it a __self__. (The term __identity owner__ was originally used for an identity controller managing itself, but this hid some of the nuance and introduced [legal concepts of ownership that are problematic](https://medium.com/@hackylawyER/do-we-really-want-to-sell-ourselves-the-risks-of-a-property-law-paradigm-for-data-ownership-b217e42edffa), so we'll avoid it here.)
+ 
+In our notation, selves are denoted with a single *upper-case* ASCII alpha, often corresponding to a first initial of their human-friendly name. For example, Alice might be represented as `A`. By preference, the first half of the alphabet is used (because "x", "y", and "z" tend to have other ad-hoc meanings). When reading aloud, the spoken form of a symbol like this is the name of the letter. The relevant [ABNF]( https://tools.ietf.org/html/rfc5234) fragment is:
```
Comment (m00sey) [inline concepts/0006-ssi-notation/README.md:55]: With the change from identity owner -> identity controller, should Identity controller be referenced here?

>In our notation, selves (or identity controllers) are denoted...

It's referenced in the section above, maybe an addition here would solidify the point?
or am I missing a nuance? I read selves and think people/institutions etc

Comment (dhh1128) [reply concepts/0006-ssi-notation/README.md]: Tweaked per @moosey's suggestion.

**Comment by dhh1128** [comment]
@TelegramSam or @swcurran : I believe we discussed this on a community call. It's stood without further comment (except for Kevin's input, which I accepted and resolved) since November. Can we merge?

**Comment by dhh1128** [comment]
approved on community B call 24 Mar 2021

**Pull request merged**